homeassistant/components/bmw_connected_drive/sensor.py

```diff
@@ -82,6 +82,17 @@ def __init__(self, account, vehicle, attribute: str, attribute_info):
         self._unique_id = f"{self._vehicle.vin}-{self._attribute}"
         self._attribute_info = attribute_info
 
+    @property
```

Comment:
This property is common to the various platforms, maybe you can define only once as a mixin ?

Reply:
Will have a look at this!

Reply:
Added a `BMWConnectedDriveBaseEntity` to init.py which contains `device_info`, `update_callback` and `async_added_to_hass`.